dlk/python/dlk/templates/include/network.tpl.h

```diff
@@ -28,6 +28,7 @@ class SYM_PUBLIC Network
     Network();
     ~Network();
 
+    int memory();
```

Comment:
To keep consistency of API, it looks better to use verb for the method name. Considering that the responsibility of this function should output debug information, how about renaming to `get_debug_info`?

Reply:
Also, I propose to change this API to return string, (or char*), so that we can see the buffer size layer by layer. (Is it possible to change the behavior of this API?)

Reply:
Yes, it's true. The name can be better and we should return more detailed information.